src/main/java/uk/gov/justice/digital/delius/transformers/CaseloadTransformer.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.digital.delius.transformers;
+
+import lombok.val;
+import uk.gov.justice.digital.delius.data.api.Caseload;
+import uk.gov.justice.digital.delius.data.api.ManagedEventId;
+import uk.gov.justice.digital.delius.data.api.ManagedOffenderCrn;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.emptySet;
+import static java.util.stream.Collectors.groupingBy;
+import static java.util.stream.Collectors.toSet;
+import static uk.gov.justice.digital.delius.data.api.CaseloadRole.OFFENDER_MANAGER;
+import static uk.gov.justice.digital.delius.data.api.CaseloadRole.ORDER_SUPERVISOR;
+
+public class CaseloadTransformer {
+    public static Caseload caseloadOf(final List<uk.gov.justice.digital.delius.jpa.standard.entity.Caseload> caseload) {
+        val groupedCaseload = caseload.stream().collect(groupingBy(entity -> entity.getRoleCode()));
+        return Caseload.builder()
+            .managedOffenders(Optional.ofNullable(groupedCaseload.get(OFFENDER_MANAGER.getRoleCode()))
+                .map(managedOffenders -> managedOffenders.stream()
+                    .map(CaseloadTransformer::managedOffenderCrnOf)
+                    .collect(toSet()))
```

Comment:
As above, mapping the useful offender details - first/last/middle names, date of birth and nomsId, as these are the displayable info for users.